src/probe/route/get-probes.ts

```diff
@@ -27,6 +27,7 @@ const handle = async (ctx: ParameterizedContext<DefaultState, DefaultContext & R
 			network: socket.data.probe.location.network,
 		},
 		resolvers: socket.data.probe.resolvers,
+		stats: socket.data.probe.stats,
```

Comment:
We agreed with @jimaek to not include this in the public data but instead:
 - [ ] add an optional `adminkey` query param to this endpoint, and include this property only if the `adminkey` is non-empty and correct,
 - [ ] correct value for `adminkey` will be set via config to some string, default is empty, which means the feature isn't available unless some value is explicitly set.

Reply:
And maybe expose the IP address to the admin too. That should simplify our debugging, no more guessing what probe is what